daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -514,7 +514,7 @@ func TestAbortCatchup(t *testing.T) {
 	abortCatchupTest(t, badCatchPoint, 400)
 }
 
-func tealCompileTest(t *testing.T, bytesToUse []byte, expectedCode int, enableDeveloperAPI bool) {
+func tealCompileTest(t *testing.T, bytesToUse []byte, expectedCode int, enableDeveloperAPI bool, params generated.TealCompileParams) {
```

Comment:
Perhaps augment this function or a make a new one to also test the returned source map?